src/Integration.UnitTests/Suppression/SonarQubeIssuesProviderTests.cs

```diff
@@ -281,36 +281,27 @@ public void GetSuppressedIssues_WhenProjectHasModulesAndIssueIsFileLevelAndIsFou
             // Arrange
             var issues = new []
             {
-                new SonarQubeIssue("foo\\foo.cs", null, "message", "sqkey:sqkey:projectId", "S2", true, SonarQubeIssueSeverity.Blocker, null, flows: null),
-                new SonarQubeIssue("foo\\FOO.cs", null, "message", "sqkey:sqkey:projectId", "S3", true, SonarQubeIssueSeverity.Blocker, null, flows: null),
-                new SonarQubeIssue("FOO\\foo.cs", null, "message", "sqkey:sqkey:projectId", "S4", true, SonarQubeIssueSeverity.Blocker, null, flows: null),
-                new SonarQubeIssue("bar\\bar.cs", null, "message", "sqkey:sqkey:projectId", "S5", true, SonarQubeIssueSeverity.Blocker, null, flows: null)
+                CreateIssue("foo\\foo.cs", "sqkey:sqkey:projectId", "S2"),
+                CreateIssue("foo\\FOO.cs", "sqkey:sqkey:projectId", "S3"),
+                CreateIssue("FOO\\foo.cs", "sqkey:sqkey:projectId", "S4"),
+                CreateIssue("bar\\bar.cs", "sqkey:sqkey:projectId", "S5")
             };
 
             SetupSolutionBinding(true,
                 issues,
                 new List<SonarQubeModule> { new SonarQubeModule("sqkey", "", ""), new SonarQubeModule("sqkey:sqkey:projectId", "", "src\\bar") });
 
-            SonarQubeIssuesProvider issuesProvider;
-
-            using (new AssertIgnoreScope())
-            {
-                issuesProvider = new SonarQubeIssuesProvider(mockSqService.Object, "sqkey", mockTimerFactory.Object, testLogger);
-                WaitForInitialFetchTaskToStart();
+            var issuesProvider = new SonarQubeIssuesProvider(mockSqService.Object, "sqkey", mockTimerFactory.Object, testLogger);
+            WaitForInitialFetchTaskToStart();
 
-                VerifyServiceGetIssues(Times.Exactly(1)); // issues should be fetched on creation
+            VerifyServiceGetIssues(Times.Exactly(1)); // issues should be fetched on creation
 
-                // Act
-                IEnumerable<SonarQubeIssue> matches;
-                // We're deliberately faking SonarQube returning paths with \ instead of / which
```

Comment:
This comment was no longer accurate, and the AssertIgnoreScope is no longer required.